backend/handlers.go

```diff
@@ -80,6 +80,21 @@ func (s *session) joinedState(cmd *proto.Packet) *response {
 			return &response{err: err}
 		}
 		return &response{packet: &proto.WhoReply{Listing: listing}}
+	case *proto.UploadCommand:
+		mediaID := make([]byte, 16)
+		if _, err := rand.Read(mediaID); err != nil {
+			return &response{err: err}
+		}
+		key := ""
+		URL := fmt.Sprintf("/media/%s/upload", mediaID)
```

Comment:
The mediaID should be human-readable. How about `%032x` instead of `%s`? It might be better to halve the length, too.